README.md

```diff
@@ -8,20 +8,46 @@ The portal site is built with [sphinx](https://www.sphinx-doc.org/).
 To build the site locally (e.g. for testing new content),
 use [conda](https://docs.conda.io/) to set up a build environment with all dependencies.
 
-First, make a local clone of this source repository on your machine.
-Then, from the `content` directory of the source repository, do this:
+First, make a local clone of this source repository on your machine. For example:
+
+```
+git clone https://github.com/ProjectPythia/projectpythia.github.io.git
+```
+
+Then, change working directories to the local repository's `content` directory:
+
+```
+cd projectpythia.github.io/content
+```
+
+Set up your a conda environment:
+
 ```
 conda env create -f ../ci/environment.yml
 conda activate pythia
 ```
 
-You can then build the site:
+You can then build the site with:
+
 ```
 make html
 ```
-and view the built site in your web browser with
+
+After building the site it may be possible to preview the portal in your web browser by simply opening the `index.html` file:
+
 ```
 open _build/html/index.html
 ```
 
+However, many of the links will not work. For all of the links found in the portal to work properly, you'll need to set up a local testing server. This can be done with Python's http.server by running the following command from within the `content` directory:
+
+```
+python -m http.server --directory _build/html/
+```
+
+and then pointing your browser at the URL: `localhost:8000`
```

Comment:
It's my understanding that this step isn't needed since we provide a `live` directive in the Makefile for running the server: 


https://github.com/ProjectPythia/projectpythia.github.io/blob/6f0a893ca6349ea963d25ed883bb2d437fba9ed2/content/Makefile#L15-L17

I think this could be replaced by 

```bash 
make live
```

Reply:
`sphinx-autobuild` has an added benefit of including a **livereload** enabled web server

Reply:
@andersy005 that would be cool. Have you tried it? When I run 'make live' I get into what looks like an infinite loop. I've tried blowing away the _build directory first.

Reply:
> When I run 'make live' I get into what looks like an infinite loop.

The `make live` command doesn't return because sphinx-autobuild keeps watching the directory for new changes (when you make changes to the source files, it rebuilds the website without you having to re-run the `make live` command) ...

Reply:
Hmm. Doesn't seem to be working for me: the links still aren't being resolved. Have you tried it? What is the order of operations you are using?

Reply:
It's currently not working on my end. I suggest merging this PR as is, then I will look into this issue in a separate PR.

Reply:
Sounds good to me if you want to resolve this comment and approve the PR. Thanks